edr_stager_dbt_project/macros/get_elementary_dbt_pkg_version.sql

```diff
@@ -32,3 +32,8 @@
     {% set database = target.project %}
     {{ return(database) }}
 {% endmacro %}
+
+{% macro databricks__target_database() %}
+    {% set database = target.schema %}
```

Comment:
I think `host` is more closely related to database.

Reply:
We've implemented it like [this](https://github.com/elementary-data/dbt-data-reliability/blob/797a5b402890a558755f0eccd767bc43f0ef52dd/macros/utils/cross_db_utils/target_database.sql#L11) on the dbt package.

Reply:
Thanks @elongl for the response.

Does that mean we have to open PR on [elementary-data/dbt-data-reliability](https://github.com/elementary-data/dbt-data-reliability/blob/797a5b402890a558755f0eccd767bc43f0ef52dd/macros/utils/cross_db_utils/target_database.sql#L11) repository as well to complement changes in elementary-action repo?